.prettierrc.js

```diff
@@ -1,7 +1,12 @@
-module.exports = {
+// @ts-check
+
+/** @type {import('prettier').Config} */
+const config = {
   singleQuote: true,
   semi: false,
   printWidth: 100,
   tabWidth: 2,
   trailingComma: 'es5',
```

Comment:
Please apply `sort-import` here. The order is

1. `@nestjs/**`
2. `react/**`
3. 3rd Party
4. 
5. `@/**` (alias absolute path for the app)
6. `@cgr/**` (for local packages)
7. ./ 
8. ../

Don't for get to separate the imports using `newline`

Or, you can create `prettier-plugin-base`, `prettier-plugin-react` and `prettier-plugin-nest` in `@cgr/project-config` to configure different import order for different based framework. 

PS. We already installed the plugin in the root package.json. But, somehow, we forgot to use the plugin. lol

Reply:
plugin is `@trivago/prettier-plugin-sort-imports`